### PR TITLE
Intel® Bug Bounty Program

### PR DESCRIPTION
**Pull request by nmanovic**
### Checklist

- [x] I submit my changes into the `develop` branch
- [ ] I have added a description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [x] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [x] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2022 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ be sure it can be reproduced in the supported version.
 ## Reporting a Vulnerability
 
 If you have information about a security issue or vulnerability in the product, please
-send an e-mail to secure@intel.com. Encrypt sensitive information using our PGP public key.
+send an e-mail to <secure@intel.com>. Encrypt sensitive information using our PGP public key.
```
Comment (nmanovic) [inline SECURITY.md:16]:
```suggestion
send an e-mail to [secure@intel.com](mailto:secure@intel.com). Encrypt sensitive information
using our PGP public key.
```

**Review by Gitgody — COMMENTED** [review]
@End

**Comment by Gitgody** [comment]
> <end
> 
> Copyright (C) 2020-2022 Intel Corporation
> 
> 
> 
> SPDX-License-Identifier: MIT
> 
> -->
> 
> 
> 
> 
> 
> 
> 
> 
> 
> 
> 
> ### Motivation and context
> 
> 
> 
> 
> 
> ### How has this been tested?
> 
> 
> 
> 
> 
> ### Checklist
> 
> 
> 
> - [x] I submit my changes into the `develop` branch
> 
> - [ ] I have added a description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
> 
> - [x] I have updated the [documentation](
> 
>   https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
> 
> - [ ] I have added tests to cover my changes
> 
> - [ ] I have linked related issues ([read github docs](
> 
>   https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
> 
> - [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
> 
> [cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))
> 
> 
> 
> ### License
> 
> 
> 
> - [x] I submit _my code changes_ under the same [MIT License](
> 
>   https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
> 
>   Feel free to contact the maintainers if that's a concern.
> 
> - [x] I have updated the license header for each file (see an example below)
> 
> 
> 
> ```python
> 
> # Copyright (C) 2022 Intel Corporation
> 
> #
> 
> # SPDX-License-Identifier: MIT
> 
> ```
> 
>

**Review by Gitgody — COMMENTED** [review]
Exit

**Pull request merged**